core/renderers/common/drawer.js

```diff
@@ -394,8 +394,9 @@ Blockly.blockRendering.Drawer.prototype.positionStatementInputConnection_ = func
     var connX = row.xPos + row.statementEdge + input.notchOffset;
     if (this.info_.RTL) {
       connX *= -1;
+    } else {
```

Comment:
This works for now--longer term, I think I want to move this into the geras-specific drawer and get rid of the dark path offset for other renderers.

Reply:
I'm down with that plan!